releasenotes/notes/expose-clc-runner-stats-dad4462c9c661f3a.yaml

```
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    The Agent can now expose runner stats via a public API.
```

```suggestion
    The Agent can now expose runner stats via a remotely-accessible authenticated API server.
```

Also, can you add some basic details on:
* whether the API is enabled by default or not, and under which conditions it's enabled by default
* which address and port it listens to by default

If any of these points are too long to explain here, please add a link to relevant docs that do explain this.

---

Also, you seem to have given this server a name (`CLC Runner API Server`), I think this name should be customer-facing since it's mentioned in the logs.